app/models/concerns/thredded/topic_common.rb

```diff
@@ -10,7 +10,7 @@ module TopicCommon
                  foreign_key: 'last_user_id'
 
       scope :order_recently_updated_first, -> { order(updated_at: :desc, id: :desc) }
-      scope :on_page, -> (page_num) { page(page_num).per(30) }
+      scope :on_page, -> (page_num) { page(page_num) }
```

Comment:
I figured `if respond_to?(:paginates_per)` was making sure the gem was loaded or something, either way having both buys us nothing.

The real question is, do we want a page size of 50 or 30?